lib/bindings/http/emitter_structured.js

```diff
@@ -1,30 +1,23 @@
 var axios = require("axios");
 
 const Constants = require("./constants.js");
+const defaults = {};
```

Comment:
Why are we creating an empty object and then adding properties to it?

```js
const defaults = {
  [Constants.HEADERS]: {
    [Constants.HEADER_CONTENT_TYPE]: Constants.DEFAULT_CE_CONTENT_TYPE
  }
};
```

Reply:
You introduced new syntax to me. I wasn't aware that you could bracket a nested property as a field name.

Reply:
OK. This is a feature of JavaScript ES2015. Here's a reference:
https://stackoverflow.com/a/19837961

Reply:
I did not know either! Looks good